README.md

```diff
@@ -3,7 +3,7 @@
 > This is Portfolio project to showcase my work as a Developer.
 
 ## Live Link
-> https://awaisanwar544.github.io/portfolio/app/index.html
+[Click here to go to the live portfolio](https://awaisanwar544.github.io/portfolio/app/index.html)
```

Comment:
- [ ] Please make sure that your link is working. Right now, the link it's showing a "404" error message, so please apply the necessary changes.

Reply:
updated to working link